virttest/openvswitch.py

```diff
@@ -459,9 +457,7 @@ def __init__(self, tmpdir, db_path=None, db_socket=None, db_pidfile=None,
         :param ovs_pidfile: Path of OVS ovs-vswitchd pid.
         :param install_prefix: Path where is openvswitch installed.
         """
-        super(man[self, OpenVSwitch], self).__init__(db_path, db_socket,
-                                                     db_pidfile, ovs_pidfile,
-                                                     dbschema, install_prefix)
+        super(man[self.__class__, OpenVSwitch], self).__init__()
```

Comment:
is any special reason to leave out those parameters ,which work as initialized value?